src/intents.cs

```diff
@@ -396,7 +414,17 @@ public enum INPhotoAttributeOptions : ulong {
 		TransferFilter = (1 << 20),
 		MonoFilter = (1 << 21),
 		FadeFilter = (1 << 22),
-		ProcessFilter = (1 << 23)
+		ProcessFilter = (1 << 23),
+		[Watch (5,0), iOS (12,0)]
+		PortraitPhoto = (1uL << 24),
+		[Watch (5,0), iOS (12,0)]
+		LivePhoto = (1uL << 25),
+		[Watch (5,0), iOS (12,0)]
+		LoopPhoto = (1uL << 26),
+		[Watch (5,0), iOS (12,0)]
+		BouncePhoto = (1uL << 27),
+		[Watch (5,0), iOS (12,0)]
+		LongExposurePhoto = (1uL << 28)
```

Comment:
Following the change, add an extra ',' at the end to simplify the next diff so that it does not happen the same as in this one.

Reply:
ahh missed this one, thanks